packages/react-nft-checkout/src/components/SwapTokensSelect/SwapTokensSelect.tsx

```diff
@@ -60,11 +61,14 @@ const SwapTokensSelect = () => {
           (error as Error)?.message ||
             'An error occurred while loading tokens. Change the network or try again later.',
         )
+      } finally {
+        fetchingTokens.current = false
       }
```

Comment:
looks like u can use isLoading instead of `fetchingTokens`

Reply:
fixed